composer.json

```diff
@@ -14,7 +14,10 @@
         "headlesslaravel/notifications": "*",
         "headlesslaravel/cards": "*",
         "headlesslaravel/metrics": "*",
-        "illuminate/console": "*"
+        "illuminate/console": "*",
+        "illuminate/routing": "*",
+        "illuminate/support": "*",
+        "composer/composer": "^2.2"
     },
```

Comment:
Do we need all these dependencies?

Reply:
We have a command in our repo, so `illuminate/console` is required.
I think we can remove `illuminate/routing`.
`illuminate/support` will be needed for different facades and service provider
`composer/composer` is needed for class mapping.

Reply:
@mitulgolakiya ok thank you